eva/parser/create_udf_statement.py

```diff
@@ -71,7 +71,12 @@ def __str__(self) -> str:
                 output_str += str(expr) + ", "
             output_str = output_str.rstrip(", ")
 
-        return f"CREATE UDF {self._name} INPUT ({input_str}) OUTPUT ({output_str}) TYPE {self._udf_type} IMPL {self._impl_path.name}"
+        if len(input_str) > 0:
```

Comment:
nit: Can probably do this in a cleaner way, by adding "INPUT" and "OUTPUT" to the strings defined above. Similarly for type.